### DISCUSSION TITLE
Example with SQL Server Connection

### DISCUSSION THREAD

**Discussion by vioarief** [gerardo-lijs/CrystalReportsRunner#23] [Q&A]
Hai, is there any example with SQL Server Connection. Will I change all configuration in my RPT file if I use this? 

Thanks Before

**Comment by gerardo-lijs** [comment]
Hi, no need to change the reports configuration. You can find the sample with SQL Server Connection in the main repo or also here -> https://github.com/gerardo-lijs/CrystalReportsRunner.Samples

**Comment by rozyjaret** [comment]
is there a way to connect to mysql? Plz reply
